packages/@react-spectrum/actionbar/docs/ActionBar.mdx

```diff
@@ -20,7 +20,7 @@ import {ActionBar, ActionBarContainer, Item} from '@react-spectrum/actionbar';
 ```
 
 ---
-category: Overlays
+category: Collection Actions
```

Comment:
I just noticed that `ActionMenu` docs were placed in "Collections". I was looking at the SideNav and I wanted an "Actions" section for `ActionGroup`, `ActionBar`, and `ActionMenu`. Maybe `ActionButton` too.

Reply:
Talked in Slack and we arrived on `Toolbars` as the section for this component.